PathOfBuildingCommunity/PathOfBuilding
# Fixes Minions with Endurance Charges still gaining resists.

Fixes #8162 .

### Description of the problem being solved:
Path of building (in the calculations tab, showing minion stats) displays the minion(s) in question has having increased elemental resistance from endurance charges, which they should no longer have
### Steps taken to verify a working solution:
- Set minion skill.
- Turn on Endurance charges.
- Check Elemental resistances breakdown.

### Link to a build that showcases this PR:
```
eNqtW1tz4rgSfp75FS7eh7sJmSKzRUgySVUy4UAyc8552RK2AO3IFmvLSdhfvy3JNgYsI2PzkBirv75J3Wq1zeiPD49abzgICfOvGp1mu2Fh32Eu8VdXjdeXuy_Dxh_fPo-miK-fl9cRoWLk2-dPI3ltUfyGKeDagOMoWGH-M-HV-xPubZDP15j5T-gvFnxn7lXjB_Nxw1og3yU8-eZQFIY_kIevGnMHwA0LhQ723cnufky4RgFyOA4ehdxxxNkTc2F0iWgIwx4i_pw5vzH_HrBoA3o1rDeC3xXRw9P0efaS0Yn4WZ3AqE-jKUVbHMw54lYIf64aY4eTNwykoNYj8QgHjohGwG7QaGkg4E60wvcZ2vYJ2usoCPkN8uDyNGa-wdhNyTrNzsWgf9G2h8Nuz-7pQNMA3y6XWFozCQifrJHvGAgrS_sUUU42lOAgo6GtQ9wfMRcrSUP8wjiiN9P5aUUUJTPw_y_C19cU_GnEV1A_rHzCsTH5lJGQ-aW0NiKeRJRCJBrRznCIgzfEiaEiE-YtiG_okyfkowkLd77uDotIpziAyOZ7iPYJwBw7DJJBFtLrNi_7nXTlG0jM56MV_UiW2JyylFUxoKw259lxOzelK834PIVmkOPMKOcsooaUfJdtulqqG_xhQPXgcwOqGf47S9ixbb3UN8blnmYU_7f30x3XbveyaduDgd3u9gf61L7ehsRB9Al9EC_yIKe-oN_Yz7DpD_ULa7XmPiQSHfhiqNX3jgT4DNiEUfcc2Bqx8AycCB0jV8C26nwV1A--YxaRr34gM2t2Ny709RLPIFLElr-g2BizExNHnMlmqYStsB9L3JqZ9Iixs_4OZdQMcWyWoHcFUb9f6FtBbORbQZjj2wL--4gSbhLAfDd1Os1eEaqkp259HKy28zXB1C1HnWg2QRuj2tD5mkUbOXxfXKlFk4WWdMkvFLhm-0VZnd5QmM24nUGxuxS52dLEUCwCwMWmRfE0YH-JspuWg40Dj0WB4YQrYiMDks1CHTZm2I0cs90pPT1cUzhhmZqRokBPSktBx5wj5_cNc1fGTpNCSiH29ZtHmw0kEbEaTBmITRDqa5IpU74MDKifYSkbRbTYLs0F7KiNBaQlgLmUA4i5LWIPL2HMjtxYRDqhT5AsPNgF5Gn5iWW2aO3kwJHK6MgjCQ3PaVP2DpqvRWskLEcNtc7uuKNVJcD-P1tj_nvkRgJufTcKRCgYyzhEGIk53WZRY8etk4OuSX_QhHK53-9ocfvtk3ZzaMMZsp38rRd11LLoXTaHF93ORbfXvhhcdotxOQ2MI8rcBoahlPx2RgnwcXOjpH15IXREXNDqOKI90eo45p3T6jC0Ye9gcTFIa_hcwuIKJhdSXE0dQQoLzELqkrqZV3wvxINyKQxvEEeWG5-Df6KAIJ93ZXM2xChw1o-Q4O8QpQvY768a2bvi2wGwk2TwUUu2oMXVg7dhAbfwh_g3RQHfJi1hSSjvAJ-QE18uDUgelDas-Zq9j903kateGKNh2kdGmw323T0eLwHGFkpKBUcoIW0UXywPhRx8p_aeUCid6V8_uMIjls9AgauGPRz2esJE0TFBwXa8T-gT0IuDrEwbvWvHHXLFSYj9NHqdPcqLT2vON-HXVuv9_b25QXzNlvgDis2mw7zWBkCg8JfwN6H0i2DbGsPnejWWH8molXAaqc552FLfxI4bENBZzWVLGCq9LjwhLn4wjkMxJm4mX0ZzISq0QpiI79gLr7cQWXdiqRzEWuxKQT3HXC2GLOaqwYMIi9lfooiK-_-JECViZtvZu4_qAYTPAi_tgAArmFlRzCmOL9uNcP348VGNjCmPmQlxyTSr6YwVsoibTHF8Uz5cGO-0niDqhFJv4js0cvGDHxcQ8SxStBCaiecp4rjvJhYd8EnFfBqBNjHxd8oWiHYTiJy_OBD3ZWdG4pszRELs_l_4GpDx85ku-GyFPbGAnjBHLgRk64GD8S3hgZZkClcSm0DfVLwJzN59KTL_bkZFqd3fmSmLrwXwRs1eY8_YTmLsgbscFvlqffjIi2NPSrZyRB-YHy9lZV-8jOMJlktZLVZxKZ0nKR78TcSlrKuGWCyKccNaQIbAyI_Vah3TCl5_-pG3EI8-1P9dsZml9Ejo_LmIlkvxWArU54F82HZ7d3c7eXn4eRtD5liezKwwWoTq8qrxk-B3qfQNzCKhoXAPpWgT4jRbyZUbW0kBV8BNUt2T9MFTPq8dgZ7T7QcOILmu4EjvBARr9UrHTyilBIoNSewhOm6iQNIzUufHCaRmtTlpPCXrOj0X8UxLa44YLMDCtoSoVnI8esITXOQuWPRkSRyxCRdPuch0iqrAL44D9bmzLZjv-PCr5yErTh0DNagHq6pRh45HC7wqK1atV9WoHn6DHaS1XQ3qwWm9w3z5eDefS0pVwOkHFKhikUPQjAkVB1XtzN5SnJLoGT7zNQ7iCkTH6QkyT0JSGDgBWURcH8YZigJfyfpY4yExpoeqtrLGBjFWkIn2ymCNQ7M0elaqP6lNZEVQ1bTQ-i9ugRRMQdz907hfjRY4IWmAauyPhwuCRObf8RsjrmqDacLlgKwoYUBxWZ2N7O1VZ3PY7KvO8Q7q89_a-Y5H9fBXTkRxlMNFFUFGTERQVeMgYqsahxcohnkU4LMZzA4rkR12VlyDpA2qXHAyWpQ54r7V2RxUd-1suGz-nY2W-R8qawwWFG4AKU1BfPDIvwFn8ILYMGQl1cpPJDvrSvFSW2GupaU5qgCPnyUX5QBFcoIR7OX3BdWiGae0gX2PERUtNkarMTx6Zl6FmWgTRRvkuwm757wafTcPht5jPASesp9zI7pRVX3oY2-bw0iv16iVnABlp0ecs-I21JwH4hD_D2Pe_64aX2y72eu37V78UQNxE2IYNx6gAL8hMI2BXH6JWEH4X-DQGQ6b7cHlRXsgPhdKvDyMx00RcZ32RPT8ohCr925-YbSBU7dAiMaDOm8qHlqi_f4GZdyCqtmbLl5nj8JadVCFg_mbiDAxpHpPrWKAEmJ1SkCuGcy9NV5swxBRS7WgLLsMA0z5Ib5bAi93JKt3hpXW_B1tDkX3z2FUwfyYR7cGHvkGDWowaFCDQWfxyDOo8urs1eCRbl3WlFlv9xi2Tl7JHznBNqisQbc0h8oie3X5365hMXRqCI9ODXr0S66EulJMt2pE9mtwYPk9o7YU26tqf7diCJdTwN1aqgNRZQ2q2qIKh_zAtitzGNQ1rZ26ckz5pVndj_3Kc2nXs6jKGD_2Iop5DZmwV0NC6VUMyn5FfKce_5-9U5ZetP3SiHOqqIpetavm6pqmpV9Xcqmtrj77CFUCNyP-qtQU5qembj1zUNtOUceBr3-eTWWdX32Pr6nYsCsrUn2LG5RNBlWTx7HAUStu9cgmlmwjyfcZmL8kq7ihpL7ELSUJSu_s9YH2X1IQrz2EryE-fgM051UI-Uqng9eMujiIWWDRhYvfm0zeh7hot08A9l9Qz7xJ0b3sn4Bmf6uawGw9Zr4Gp87FSwLh0QsbWhnxe_OM0hnys_pdFBg2d5jsJBqLCsmK0OelfCIAMOl1A5iX_vhWvJ2JA-xKmRPxCs0c02WGiX3CziQ7JIB-37485Zv0sWMCGvaSn2YWztnhVPfStZ2uU9Wild--fR61Dn-O_i-oagsv
```
### Before screenshot:
![image](https://github.com/user-attachments/assets/bf8a7f49-55a7-4bf4-a437-bda23931b026)
### After screenshot:
![image](https://github.com/user-attachments/assets/b7b52464-4bae-4b70-8012-2a6ee99a2c2d)
